Profiler_Test/Proxies/TiaProfiler.cs

```diff
@@ -0,0 +1,134 @@
+
+using Cqse.Teamscale.Profiler.Commons.Ipc;
+using NUnit.Framework;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+
+namespace Cqse.Teamscale.Profiler.Dotnet.Proxies
+{
+    public class TiaProfiler : Profiler
+    {
+        private readonly IpcConfig ipcConfig;
+
+        public TiaProfiler(DirectoryInfo basePath, DirectoryInfo targetDir, IpcConfig ipcConfig) : base(basePath, targetDir)
+        {
+            this.ipcConfig = ipcConfig;
+            this.LightMode = true;
+        }
+            
+        public override void RegisterOn(ProcessStartInfo processInfo, Bitness? bitness = null)
+        {
+            base.RegisterOn(processInfo, bitness);
+            
+            processInfo.Environment["COR_PROFILER_TIA"] = "true";
+            processInfo.Environment["COR_PROFILER_TIA_SUBSCRIBE_SOCKET"] = ipcConfig.PublishSocket; // PUB-SUB
+            processInfo.Environment["COR_PROFILER_TIA_REQUEST_SOCKET"] = ipcConfig.RequestSocket; // REQ-REP
+        }
+
+        public TiaTestResult Result
+        {
+                get
+                {
+                    FileInfo actualTrace = GetSingleTraceFile();
+                    TiaTestResult testResult = new TiaTestResult(File.ReadAllLines(actualTrace.FullName));
```

Comment:
Should we reuse the logic for parsing trace files from the upload daemon here? It seems a bad idea to duplicate that parsing logic, even if in the tests.